Mail2Bug/Email/EWS/EWSIncomingMessage.cs

```diff
@@ -2,22 +2,27 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Text;
 using log4net;
 using Microsoft.Exchange.WebServices.Data;
 
 namespace Mail2Bug.Email.EWS
 {
     public class EWSIncomingMessage : IIncomingEmailMessage
     {
+        private const int PidTagBodyHtmlTag = 0x1013;
```

Comment:
Please add a comment with a reference to where this constant is taken from (i.e. name of the source + link)